pom.xml

```diff
@@ -109,25 +114,6 @@
                     </execution>
                 </executions>
             </plugin>
-                <plugin>
-                    <groupId>org.jacoco</groupId>
```

Comment:
This code can't be deleted from pom.xml. Make sure to checkout the latest pom.xml from main branch with:

`git checkout main pom.xml`

Then make your changes adding dependency for jackson-databind

Reply:
Solved